packager/media/file/udp_file_posix.cc

```diff
@@ -187,6 +187,9 @@ bool UdpFile::Open() {
   local_sock_addr.sin_family = AF_INET;
   local_sock_addr.sin_port = htons(dest_port);
   local_sock_addr.sin_addr.s_addr  = htonl(dest_addr);
+  // We do not need to require exclusive bind of udp sockets
+  int optval = 1;
+  setsockopt(new_socket.get(), SOL_SOCKET, SO_REUSEADDR, &optval, sizeof(optval));
```

Comment:
Please check the return value of setsockopt. Log an error and return false if it fails.

I am not familiar with SO_REUSEADDR, just for my own knowledge, does it matter if we call setsockopt before or after bind?